app/controllers/application_controller.rb

```diff
@@ -46,6 +46,11 @@ def deny_team_in_top_ten
     redirect_back fallback_location: user_root_path, alert: I18n.t('teams.in_top_ten')
   end
 
+  def deny_if_not_admin
+    return if current_user&.admin?
+    raise ActiveRecord::RecordNotFound
+  end
+
```

Comment:
Can we use it in any other place?

Reply:
I looked around a bit when writing it and unfortunately couldn't find anywhere else.